src/widgets/notes/notes-overview.component.tsx

```diff
@@ -21,124 +21,129 @@ export default function NotesOverview(props: NotesOverviewProps) {
     if (patient) {
-      const sub = getEncounterObservableRESTAPI(
-        patientUuid
-      ).subscribe((response: any) => setPatientNotes(response.results));
+      const sub = getEncounterObservableRESTAPI(patientUuid).subscribe(
+        (response: any) => setPatientNotes(response.results),
+        createErrorHandler()
+      );
       return () => sub.unsubscribe();
     }
   }, [patient, patientUuid]);
 
   function fhirNotesOverview() {
     return (
-      <SummaryCard name="Notes">
-        <table className={styles.tableNotes}>
+      <SummaryCard
+        name={t("Notes", "Notes")}
+        styles={{ width: "100%" }}
+        link={notesPath}
+      >
+        <table className={`omrs-type-body-regular ${styles.notesTable}`}>
           <thead>
-            <tr className={styles.tableNotesRow}>
-              <th className={`${styles.tableNotesHeader} ${styles.tableDates}`}>
-                Date
-              </th>
-              <th className={styles.tableNotesHeader}>
-                Encounter type, Location
-              </th>
-              <th className={styles.tableNotesHeader}>Author</th>
+            <tr className={styles.notesTableRow}>
+              <th>Date</th>
+              <th style={{ textAlign: "left" }}>Encounter type, Location</th>
+              <th>Author</th>
               <th></th>
             </tr>
           </thead>
           <tbody>
             {patientNotes &&
               patientNotes.slice(0, 5).map(note => (
-                <tr key={note.id} className={styles.tableNotesRow}>
-                  <td className={styles.tableNotesDate}>
-                    {formatNotesDate(note?.location[0].period.end)}
+                <tr key={note.id} className={styles.notesTableRow}>
+                  <td className={styles.noteDate}>
+                    {formatNotesDate(note?.location[0]?.period?.end)}
                   </td>
-                  <td className={styles.tableNotesData}>
-                    {note?.type[0]?.coding[0]?.display || "\u2014"}
-                    <div className={styles.location}>
+                  <td className={styles.noteInfo}>
+                    <span>{note?.type[0]?.coding[0]?.display || "\u2014"}</span>
+                    <div>
                       {note?.location[0]?.location?.display || "\u2014"}
                     </div>
                   </td>
                   <td className={styles.tableNotesAuthor}>
                     {getAuthorName(note) || "\u2014"}
                   </td>
-                  <td
-                    className={styles.tdLowerSvg}
-                    style={{ textAlign: "end" }}
-                  >
-                    <svg
-                      className="omrs-icon"
-                      fill="var(--omrs-color-ink-low-contrast)"
-                    >
-                      <use xlinkHref="#omrs-icon-chevron-right" />
-                    </svg>
+                  <td>
+                    <Link to={`${match.path}/${note.uuid}`}>

```

Comment:
This should be notesPath, not match.path

Reply:
Fixed